### PR TITLE
[11.0] [IMP] crm_claim: Multi-company

### PR DESCRIPTION
**Pull request by pedrobaeza**
Include record rule for multi-company.

@Tecnativa

### CODE REVIEW

File: crm_claim/__manifest__.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
         'views/res_partner_views.xml',
         'views/crm_claim_menu.xml',
         'security/ir.model.access.csv',
+        'security/crm_claim_security.csv',
```
Comment (oihane) [inline crm_claim/__manifest__.py]: :open_mouth: mistyped?? it should be `*.xml` not `*.csv`

Comment (pedrobaeza) [reply crm_claim/__manifest__.py]: Oops, fixed

**Review by oihane — APPROVED** [review]
Technically looks good, runbot is not working to functionally test it.

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test

**Comment by pedrobaeza** [comment]
Thanks for reviewing.

**Comment by pedrobaeza** [comment]
Cherry-picked for v12 in 9232271c423d22cce8e0f96b46c62cbe04aeb91e

**Pull request merged**